TAS_TSDAdapter/TASTSDAdapter.cs

```diff
@@ -48,10 +49,10 @@ public partial class TasTSDAdapter : BHoMAdapter
         [Input("hour", "Hour between 1 and 24 inclusive for Hourly Results")]
         [Input("day", "Day between 1 and 365 inclusive for Daily results")]
         [Input("tasSettings", "Input additional settings the adapter should use.")]
-        [Output("adapter","adapter for TAS tSD")]
-        public TasTSDAdapter(string tSDFilePath = "", TSDResultType tsdResultQuery = TSDResultType.Simulation, SimulationResultType simType = SimulationResultType.BuildingResult, ProfileResultUnit resultUnit = ProfileResultUnit.Yearly, ProfileResultType resultType = ProfileResultType.TemperatureExternal, int hour = -1, int day = -1, TASSettings tasSettings = null)
+        [Output("adapter", "adapter for TAS tSD")]
+        public TasTSDAdapter(FileSettings fileSettings, TSDResultType tsdResultQuery = TSDResultType.Simulation, SimulationResultType simType = SimulationResultType.BuildingResult, ProfileResultUnit resultUnit = ProfileResultUnit.Yearly, ProfileResultType resultType = ProfileResultType.TemperatureExternal, int hour = -1, int day = -1, TASSettings tasSettings = null)
         {
-            tsdFilePath = tSDFilePath;
+            tsdFilePath = fileSettings.FileName;
```

Comment:
This should be `tsdFilePath = filesettings.GetFullFileName();`

Probably need to add `using BH.Engine.Adapter;` to the top as well.

Reply:
Input documentation also needs updating on line 44 - I would recommend:
`[Input("fileSettings", "Input the file settings to get the file name and directory the TAS TSD Adapter should use")]